src/Sylius/Bundle/PaymentBundle/DependencyInjection/SyliusPaymentExtension.php

```diff
@@ -42,5 +45,14 @@ public function load(array $config, ContainerBuilder $container)
         }
 
         $container->setParameter('sylius.payment_gateways', $config['gateways']);
+
+        $factoryDefinition = new Definition(Factory::class);
+        $factoryDefinition->setArguments([new Parameter('sylius.model.payment.class')]);
+        $paymentFactoryDefinition = $container->getDefinition('sylius.factory.payment');
+        $paymentFactoryClass = $paymentFactoryDefinition->getClass();
+        $decoratedPaymentFactoryDefinition = new Definition($paymentFactoryClass);
+        $decoratedPaymentFactoryDefinition->setArguments([$factoryDefinition]);
```

Comment:
How about squeezing this to one line:
`$decoratedPaymentFactoryDefinition = new Definition($paymentFactoryClass, [$factoryDefinition]);`?